app/views/helpers/category/update.phtml

```diff
@@ -36,6 +36,56 @@
 				</div>
 			</div>
 
+			<div class="form-group">
+				<label class="group-name" for="default_sort"><?= _t('sub.category.preferred_sort')?> </label>
+				<div class="group-controls">
+					<select name="default_sort_option" id="default_sort_category" class="w50">
+						<option value="id|DESC" <?=
+							$this->category->get_default_sort() === 'id' && $this->category->get_default_order() === 'DESC' ? 'selected="selected"' : ''  ?>><?=
+							_t('sub.category.preferred_sort.id_desc') ?></option>
+						<option value= "date|DESC" <?=
```

Comment:
Instead of `|`, I suggest to divide this option in two, like in the configuration of the reading view